mp/src/game/server/momentum/mom_timer.cpp

```diff
@@ -104,6 +110,11 @@ bool CMomentumTimer::Start(CMomentumPlayer *pPlayer)
         // We allow cheats to be enabled but we should warn the player that times won't submit
         DispatchCheatsMessage(pPlayer);
     }
+    if (TickSet::GetCurrentTickrate() != TickSet::s_DefinedRates[TickSet::TICKRATE_66])
```

Comment:
This is problematic because gamemodes can override the tickrate, and modes like Bhop and Tricksurf default to 100 tick. You will want to instead compare:
```suggestion
    if (!CloseEnough(g_pGlobals->interval_per_tick, g_pGameModeSystem->GetGameMode()->GetTickInterval())
```